src/sentry/conf/server.py

```diff
@@ -1132,7 +1139,7 @@ def create_partitioned_queues(name):
 )
 
 # Delay (in ms) to induce on API responses
-SENTRY_API_RESPONSE_DELAY = 0
+SENTRY_API_RESPONSE_DELAY = 150 if IS_DEV else None
```

Comment:
Yeah, so what I'd do then is if you'r eok with that, `sentry devserver` can set this `SENTRY_DEVELOPMENT` environment variable or whatever automatically, then keep the same logic here in sentry.conf.py. This way devserver will explicitly enable it instead of detecting that you're running on macOS. Then it'd also still allow being overridden in my own config, etc.